nvflare/private/fed/client/fed_client_base.py

```diff
@@ -61,6 +70,7 @@ def __init__(
         overseer_agent=None,
         args=None,
         components=None,
+        cell: Cell = None,
```

Comment:
can the cell be None? should it be required arguments?

Reply:
cell can be created by SP switching as well. No need to be a required argument.